package.json

```diff
@@ -1,9 +1,10 @@
 {
-  "name": "bip32-ed25519-ts",
+  "name": "xhd-wallet-api",
   "version": "0.0.1",
   "description": "A Typescript implementation of BIP-32 Ed25519 for Algorand ARC-52.",
-  "main": "index.js",
+  "main": "dist/index.js",
```

Comment:
Since the artifacts are ESM, I would not use `main` and prefer the `exports` key. 

You can also safely set `"type": "module"`